packages/simple_trail/.rubocop_todo.yml

```diff
@@ -0,0 +1,22 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config --exclude-limit 500`
+# on 2023-02-03 16:18:39 UTC using RuboCop version 1.20.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: AllowedMethods.
+# AllowedMethods: enums
+Lint/ConstantDefinitionInBlock:
+  Exclude:
+    - 'spec/models/simple_trail/history_spec.rb'
```

Comment:
can you try to address these violations? our goal with new packages is to not need the todo file at all -- it's mentioned in the documentation on power-tools, and these two may be pretty easy to resolve.

Reply:
I did fix one of the issues that was causing the `rubocop` issue, but there is still one. I will come back to this later.